README.md

```diff
@@ -36,6 +41,18 @@
 - LinkedIn: [@oyelakin](https://www.linkedin.com/in/oyelakin-ridwan-4b4a02b6/)
 
 
+
+## :handshake: Contributing
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)[Not available]
+
+
```

Comment:
- [ ] your project functionality is working well on localhost, although please ensure to deploy this project using either Heroku, Netlify, or Github pages.

Reply:
Already deployed but due to the result of a merge conflict. It takes time to show. It's there now. Thanks

Reply:
This issue is well resolved! ✔️